extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputFormat.java

```diff
@@ -59,6 +70,17 @@ public boolean isSplittable()
   @Override
   public InputEntityReader createReader(InputRowSchema inputRowSchema, InputEntity source, File temporaryDirectory)
   {
+    KafkaRecordEntity kafkaInputEntity = (KafkaRecordEntity) source;
```

Comment:
we should have a fallback in case the the input entity is not a kafka record, and not fail in this case.

Reply:
updated, and that fixed the local testing against `telemetry.metrics.otel` topic ingestion.